source/support.tex

```diff
@@ -6019,8 +5932,7 @@
 the definition in~\ref{expr.call} applies.
 \item
 The restrictions that C places on the second parameter to the
-\indexlibraryglobal{va_start}%
-\tcode{va_start} macro in header \libheader{stdarg.h}
+\tcode{\libmacro{va_start}} macro in header \libheader{stdarg.h}
```

Comment:
Drop surrounding \tcode, because it's redundant with \libmacro

And please do grep for this situation and fix all offenders; it happens often.